docker_services_cli/cli.py

```diff
@@ -107,15 +139,18 @@ def cli(ctx, filepath, verbose):
     type=int,
     help="Number of times to retry a service's healthcheck.",
 )
+@services_by_service_type
 @env_output(env_set_command="export")
 @click.pass_obj
 def up(services_ctx, services, no_wait, retries):
-    """Boots up the required services."""
-    _services = list(services)
-
-    if not _services:
-        click.secho("No service was provided... Exiting", fg="red")
-        exit(0)  # Do not fail to allow SQLite
+    """Boots up the required services.
+
+    Example:
+        $ docker-services-cli up --db postgresql11
```

Comment:
Can we append a line break here?

It shows like this:

```console
Usage: docker-services-cli up [OPTIONS]

  Boots up the required services.

  Example:     $ docker-services-cli up --db postgresql11 Note: All services
  will be boot up if no service is specified.
```

Reply:
Fixed, thanks!:
```console
$ docker-services-cli up --help
Usage: docker-services-cli up [OPTIONS]

  Boots up the required services.

  Example:

       $ docker-services-cli up --db postgresql11

  Note: All services will be boot up if no service is specified.
```